pkg/chunk/schema_config.go

```diff
@@ -315,7 +315,7 @@ func (cfg *SchemaConfig) Load() error {
 // PrintYaml dumps the yaml to stdout, to aid in migration
 func (cfg SchemaConfig) PrintYaml() {
```

Comment:
This function is never called. We may drop it (in case will be required in the future, it's fairly easy to rewrite it).

Reply:
Removed in a90e15d